views/js/controller/Delivery/monitoring.js

```diff
@@ -1055,6 +1048,11 @@ define([
                                 });
                             }
                             loadingBar.stop();
+
+                            appController.on('change', function () {
```

Comment:
@jsconan I've got new issue, that I can't to do `off('chage')` and now each polling.stop will be accumulated. Maybe we have something like `once` to delete it after it worked

Reply:
@krampstudio ? :)

Reply:
thank you!